framework/runtime.cc

```diff
@@ -24,34 +17,40 @@ namespace opensn
 // Global variables
 Logger& log = Logger::GetInstance();
 mpi::Communicator mpi_comm;
+bool use_caliper = false;
+std::string cali_config("runtime-report(calc.inclusive=true),mem.pages,,max_column_width=80");
+cali::ConfigManager cali_mgr;
 Timer program_timer;
-
-std::vector<std::shared_ptr<MeshContinuum>> mesh_stack;
 int current_mesh_handler = -1;
+bool suppress_color = false;
+std::filesystem::path input_path;
 
+std::vector<std::shared_ptr<MeshContinuum>> mesh_stack;
 std::vector<std::shared_ptr<SurfaceMesh>> surface_mesh_stack;
 std::vector<std::shared_ptr<FieldFunctionInterpolation>> field_func_interpolation_stack;
 std::vector<std::shared_ptr<UnpartitionedMesh>> unpartitionedmesh_stack;
-
 std::vector<std::shared_ptr<Material>> material_stack;
 std::vector<std::shared_ptr<MultiGroupXS>> multigroup_xs_stack;
 std::vector<std::shared_ptr<FieldFunction>> field_function_stack;
-
 std::vector<std::shared_ptr<AngularQuadrature>> angular_quadrature_stack;
-
 std::vector<std::shared_ptr<Object>> object_stack;
 std::vector<std::shared_ptr<SpatialDiscretization>> sdm_stack;
 std::vector<std::shared_ptr<PostProcessor>> postprocessor_stack;
 std::vector<std::shared_ptr<Function>> function_stack;
 
-bool suppress_color = false;
-std::filesystem::path input_path;
-
 int
 Initialize()
 {
-  auto& t_main = log.CreateTimingBlock(opensn::name);
-  t_main.TimeSectionBegin();
+  if (use_caliper)
+  {
+    cali_mgr.add(cali_config.c_str());
+    cali_set_global_string_byname("opensn.version", GetVersionStr().c_str());
+    cali_set_global_string_byname("opesn.input", input_path.c_str());
+    cali_mgr.start();
+  }
+
+  CALI_MARK_BEGIN("Simulation");
```

Comment:
Would it make sense to mark this section "OpenSn simulation"? If so, then use `opensn::name + " Simulation"`. 

In case OpenSn runs coupled with another code using caliper and the code also used "Simulation", would those 2 sections be merged into single chunk? Just asking...